tests/std/tests/Dev11_0485243_condition_variable_crash/test.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+// REQUIRES: debug_CRT

```

Comment:
This file tests several fixes, only one of which was a memory leak. I believe that the changes to this file should be reverted in order to preserve coverage.